src/integrationtests/java/com/aws/greengrass/integrationtests/DiskSpoolIntegrationTest.java

```diff
@@ -243,8 +244,7 @@ void GIVEN_disk_spool_plugin_WHEN_operation_fails_with_database_corruption_THEN_
 
         // Corrupt Database
         try (RandomAccessFile f = new RandomAccessFile(spoolerDatabaseFile.toFile(), "rw")) {
-            f.seek(100);
-            f.writeBytes("Garbage");
+            f.writeBytes(Utils.generateRandomString(256));
```

Comment:
don't we want to write specifically to a particular location? eg. 100 bytes in?

Reply:
maybe, for whatever reason this PR broke that test, and haven't been able to figure out how to get it to corrupt again, even after looking through sqllite's corruption tests.  in that testing, i found that corrupting the beginning of the file makes it a different "not a db" error, which i added to our corruption handling as well

Reply:
and by "broke" i mean, corruption is not detected and insert completes successfully